.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -8,106 +8,110 @@ assignees: ''
-
-       * https://github.com/confidential-containers/attestation-service
-       * Cut a release (AS/RVPS images will be automatically built triggered by release)
-
-- [ ] 8. Cut a guest-components v<TARGET_RELEASE> release
-
-- [ ] 9. Cut a td-shim v<TARGET_RELEASE> release, if changes happened in the project
-
-- [ ] 10. Update kbs to use the tagged attestation-service and guest-components, cut a release and make image
-
-       * https://github.com/confidential-containers/kbs/blob/main/src/api/Cargo.toml
-         * Change the revision for the `as-types` and `attestation-service` crates (both use `v<TARGET_RELEASE>`) and update the lock file
-       * https://github.com/confidential-containers/kbs/blob/main/tools/client/Cargo.toml
-         * Change the revision for the `as-types` and `kbs_protocol` crates (both use `v<TARGET_RELEASE>`)
-       * Cut a release
-         * kbs image will be automatically built triggered by release, so ensure that the [release workflow](https://github.com/confidential-containers/kbs/actions/workflows/release.yaml) ran successfully
-
-- [ ] 11. Update Enclave CC to use the released version of image-rs
-
-    * redo step 1, but now using v<TARGET_RELEASE>
-
-- [ ] 12. Update Kata Containers to the latest released version of:
-
-    * image-rs and attestation-agent (redo step 2, but now using the v<TARGET_RELEASE>)
-
-- [ ] 13. Update the operator to use the images generated from the latest commit of both Kata Containers and Enclave CC
-
-    * redo step 5, but now targetting the latest payload image generated for Kata Containers and Enclave CC
-
-- [ ] 14. Make sure all the operator tests are passing
-
-- [ ] 15. Cut an Enclave CC release
-
-- [ ] 16. Add a new Kata Containers tag
-
-- [ ] 17. Wait for release kata-runtime-payload to be successfully built
-    * After the Kata tag is created wait for (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload.yaml) to be successfully completed, so the latest commit kata-runtime-payload for the release is created
-
-- [ ] 18. Update peer pods to use the release versions and then cut a release following the [documented flow](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#cutting-releases)
-
-## Release
-
-
-- [ ] 19. Update the operator to use the release tags coming from Enclave CC and Kata Containers
-
-    * redo step 5, but now targeting the latest release of the payload image generated for Kata Containers eand Enclave CC
-
-- [ ] 20. Update the Operator version
+    **Also, update the [operator version](https://github.com/confidential-containers/operator/blob/main/config/release/kustomization.yaml#L7)**
+   
+- [ ] 5. :wrench: **Update peer pods to use the release versions and then cut a release following the [documented flow](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#cutting-releases)**
```

Comment:
I think there was some work done to get peerpods as part of the operator. Is this part still valid for v0.9.0?

Reply:
I don't have the full picture here. @stevenhorsman has been working on https://github.com/confidential-containers/cloud-api-adaptor/pull/1754 to catch up to main. I'm not sure if the instructions linked here are still up to date or if they should be pulled more clearly into this doc

Reply:
I was referring to https://github.com/confidential-containers/operator/pull/349 which to me suggests that all of peer-pod can also be set up with an operator kustomization overlay.

Reply:
Yeah, so with the operator changes there is a different user path for peer pods:
- In dev we need to install the CoCo operator with the peer pod extras `kubectl apply -k "github.com/confidential-containers/operator/config/overlays/peerpods/default"` rather than `kubectl apply -k "github.com/confidential-containers/operator/config`, so there is an open question I raised with @bpradipt about whether we need a new peer pods kustomize in the release directory as well, so we'd do `kubectl apply -k "github.com/confidential-containers/operator/config/release/peerpods`
directory that would have resources of `../overlays/peerpods/` and set the tag, but I'm not sure that you can double update kustomizes, so might could also not bother with a release, but point to `github.com/confidential-containers/operator/config/overlays/peerpods/default?ref=<release>` instead and ensure that it is pinned to the release tag of the [quay.io/confidential-containers/operator](http://quay.io/confidential-containers/operator) image though.

I think it also depends if we want peer pods to be part of the 0.9.0 release as that would mean that https://github.com/confidential-containers/cloud-api-adaptor/pull/1754 is in as well, which I have basically working, but currently requires a bunch of hacks and manual workarounds (partly to resolve the snapshotter caching issue)

Reply:
Something else we might need to consider is now that the operator links to some of the peer pods code in https://github.com/confidential-containers/operator/blob/main/config/overlays/peerpods/crd/kustomization.yaml, should we be releasing peer pods and then bumping that before the operator release. @bpradipt can probably confirm if that is the intention. If so we need to get peer pods higher up this list?

Oh - I've just realised that we've introduced a cyclic dependency between the operator and peer pods releases, so this is going to take some more thought 😿

Reply:
> Oh - I've just realised that we've introduced a cyclic dependency between the operator and peer pods releases

@stevenhorsman what's the dependency from peer-pods to the operator?

Reply:
We use the operator in out test and deploy scripts to install CoCo and as part of our peer pods release process at the moment we bump it to use the released version of the operator as we were always the last things to get done, so we'll need to re-think that

Reply:
I had a chat with Pradipta about this and we think that the simplest thing here is that in the peer pods usage of the operator, for releases we can just pin the latest operator hash as it's mainly used for development & tests, then we should move the peer pods release to be done before the operator release like the other projects and we can just pull that release version into the operator's kustomize. This should remove the CAA dependency on the operator.

Reply:
Ok, that sounds good. As per the discussion this morning it sounds like we won't have v0.9.0 for CAA so I have removed this step. For v0.10.0 we can add it back and move it forward as needed. Sound good?

Reply:
Yep - makes that works for me. Thanks!